Directory.Build.props

```
@@ -153,7 +153,8 @@
     <_runtimeOS Condition="$(_runtimeOS.StartsWith('tizen'))">linux</_runtimeOS>
     <_runtimeOS Condition="'$(PortableBuild)' == 'true'">$(_portableOS)</_runtimeOS>
 
-    <_packageOS Condition="'$(CrossBuild)' == 'true'">$(_hostOS.ToLowerInvariant)</_packageOS>
+    <_packageLibc Condition="'$(_packageLibc)' == '' and $(_runtimeOS.Contains('musl'))">-musl</_packageLibc>
```

nit: we normally do not override private properties (prefixed with `_`) from command-line / import-site.
I think here we do not intend to override this value from command-line:
```suggestion
    <_packageLibc Condition="$(_runtimeOS.Contains('musl'))">-musl</_packageLibc>
```